.gcal.php

```diff
@@ -0,0 +1,85 @@
+<?php
+require_once 'vendor/autoload.php';
+require_once '.functions.php';
+
+putenv('GOOGLE_APPLICATION_CREDENTIALS=gcal_creds.json');
+
+Class GoogleCalendar {
+
+    private $service;
+    private $calendar_id = 'rpiambulance.com_uuc6tmlmf4ct8qeps19oj1i69g@group.calendar.google.com';
```

Comment:
1. Please change this to the calendar I sent you on Slack
1. Please place this variable in an environment file or otherwise so it stays private and not stored in Git

Reply:
This is a publicly available id and therefore does not have to be private.

Reply:
![image](https://user-images.githubusercontent.com/5694673/64183521-02aea080-ce38-11e9-9c11-9a18291fd171.png)

I gave you the secret one. The calendar is not and should not be made public to everyone.

Reply:
Available how?

Reply:
When you view the calendar. Or in the event it is the primary calendar of the user it is just their email.